README.md

```diff
@@ -47,33 +47,63 @@ This project is used to manage IAM permissions for
-| assume_domainmanager_provisionaccount_policy_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the Domain Manager accounts. | `string` | `DomainManager-AssumeProvisionAccount` | no |
-| assume_sharedservices_provisionprivatednsrecords_policy_name | The name to assign the IAM policy that allows assumption of the role that allows access to provision DNS records in private zones in the Shared Services account. | `string` | `SharedServices-AssumeProvisionPrivateDNSRecords` | no |
-| aws_region | The AWS region to deploy into (e.g. us-east-1) | `string` | `us-east-1` | no |
-| provisioner_users_group_name | The name of the group to be created for provisioner users. | `string` | `domain_manager_provisioners` | no |
+| assume\_access\_dm\_terraform\_backend\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources. | `string` | `"The IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources."` | no |
+| assume\_access\_dm\_terraform\_backend\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows access to Domain Manager-related Terraform backend resources. | `string` | `"AssumeAccessDomainManagerTerraformBackend"` | no |
+| assume\_domainmanager\_provisionaccount\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the Domain Manager accounts. | `string` | `"The IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the Domain Manager accounts."` | no |
+| assume\_domainmanager\_provisionaccount\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to provision all AWS resources for Domain Manager in the Domain Manager accounts. | `string` | `"DomainManager-AssumeProvisionAccount"` | no |
+| assume\_sharedservices\_provisionprivatednsrecords\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows access to provision DNS records in private zones in the Shared Services account. | `string` | `"SharedServices-AssumeProvisionPrivateDNSRecords"` | no |
+| aws\_region | The AWS region to deploy into (e.g. us-east-1) | `string` | `"us-east-1"` | no |
+| provisioner\_users\_group\_name | The name of the group to be created for provisioner users. | `string` | `"domain_manager_provisioners"` | no |
 | tags | Tags to apply to all AWS resources created | `map(string)` | `{}` | no |
```

Comment:
This would need to be changed in the variable description as well. Mirrors changes made upstream.

```suggestion
| tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
```

Reply:
Fixed in commit 93b6324.